src/components/PageSettingsModal/ResourcePageSettingsModal.tsx

```diff
@@ -185,6 +177,13 @@ export const ResourcePageSettingsModal = ({
         })
         setValue("permalink", "")
       }
+      if (pageData.content.frontMatter.layout === "external") {
+        // remove https:// from resource pages with external permalinks
+        setValue(
+          "permalink",
+          pageData.content.frontMatter.permalink.replace("https://", "")
```

Comment:
Just to clarify here - why do we need to remove the `https://` ah?

Reply:
This is for the display of the field - we're autopopulating the `https://` for users, so it's stored as `https://sitename.com`. When we retrieve the permalink, we should also only display `sitename.com`, hence the removal